apps/cnquery/cmd/plugin.go

```diff
@@ -197,7 +197,7 @@ func (c *cnqueryPlugin) RunQuery(conf *run.RunQueryConfig, runtime *providers.Ru
 		if conf.Format != "json" {
 			sh.PrintResults(code, results)
 		} else {
-			reporter.BundleResultsToJSON(code, results, out)
+			_ = reporter.CodeBundleToJSON(code, results, out)
```

Comment:
why discard the result here?

Reply:
we only discard the error. output is rendered into out writer